scripts/ci/jobs/ocp_vm_scanning_e2e_tests.py

```
@@ -0,0 +1,84 @@
+os.environ["INSTALL_CNV_OPERATOR"] = "true"
+os.environ["ROX_VIRTUAL_MACHINES"] = "true"
+os.environ["ROX_SCANNER_V4"] = "true"
+# TODO(ROX-34355): Move images to quay.io/rhacs-eng/ before merging to main.
+os.environ["VM_IMAGE_RHEL9"] = "quay.io/prygiels/rhel9-dnf-primed:latest"
+os.environ["VM_IMAGE_RHEL10"] = "quay.io/prygiels/rhel10-dnf-primed:latest"
+
+
+class VMScanningPostTest(PostClusterTest):
+    """Extends standard post-test with CNV namespace logs and VM artifacts."""
+
+    VM_ARTIFACTS_DIR = "/tmp/vm-scanning-artifacts"
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/vm-scanning-artifacts"

(S108)